benchmark/README.md

```diff
@@ -86,18 +86,29 @@ Using the Benchmark Driver
 * `--num-samples`
     * Control the number of samples to take for each test
 * `--list`
-    * Print a list of available tests
+    * Print a list of available tests matching specified criteria
+* `--tags`
+    * Run tests that are labeled with specified [tags]
+    (https://github.com/apple/swift/blob/master/benchmark/utils/TestsUtils.swift#L19)
+    (coma separated list)
```

Comment:
Do you want to make it clear that a benchmark requires ALL tags?

Reply:
Gladly, I'm just having trouble formulating proper english sentence here… How about:
> * Run tests that are labeled with specified [tags](https://github.com/apple/swift/blob/master/benchmark/utils/TestsUtils.swift#L19) (coma separated list); Multiple tags are interpreted as logical AND, i.e. run only test that are labeled with all the supplied tags
?

Reply:
That sounds fine to me.